pkgs/crypto/keyshares.go

```diff
@@ -85,11 +85,11 @@ func ValidateKeysharesCLI(ks *wire.KeySharesCLI, operators []*wire.Operator, own
 func VerifyValidatorAtSharesData(ids []uint64, keyShares, expValPubKey []byte) error {
 	// Verify that IDs are unique and ordered.
 	if !sort.SliceIsSorted(ids, func(i, j int) bool { return ids[i] < ids[j] }) {
-		return fmt.Errorf("operators and not unique and ordered")
+		return fmt.Errorf("operators not unique or not ordered")
```

Comment:
this is only "not ordered"

Reply:
ids[i] < ids[j] -> strict less. Same IDs will result in error, but yes we check order then ill change error